config_iter_sdcc.sh

```diff
@@ -0,0 +1,27 @@
+# Start from clean environment
+module purge
+
+# IMAS and iWrap
+module load IMAS iWrap
+
+# Compiler
+export F90=ifort
+export FC=ifort
+
+# For debugging, just in case
+module load TotalView
+
+# Actor folder
+export ACTOR_FOLDER=~/public/PYTHON_ACTORS
+mkdir -p $ACTOR_FOLDER
+
+# Need to remove the stack limit to avoid segmentation fault inside codes
+ulimit -Ss unlimited
+
+# Libraries needed for code parameters
+module load XMLlib/3.3.1-intel-2020b
+
+# EXTEND PYTHON PATH AND AVOID DOUBLONS
+export PYTHONPATH=$ACTOR_FOLDER:$PYTHONPATH
+export PYTHONPATH="$(perl -e 'print join(":", grep { not $seen{$_}++ } split(/:/, $ENV{PYTHONPATH}))')"
+
```

Comment:
Is ITER using Lmod to manage modules? If so I would convert that into a module. We could put ECRad into a public folder and then make it available to people with
```
module load ECRad
```

Reply:
I don't know exactly how modules are managed on sdcc, but clearly the plan is to have a module for each code. So having an ECRad module makes sense. We have for example a CASPER module, which is another synthetic diagnostic.